hazelcast/src/main/java/com/hazelcast/map/impl/operation/MergeOperation.java

```diff
@@ -70,8 +70,7 @@ public Object getResponse() {
 
     @Override
     public boolean shouldBackup() {
-        final Record record = recordStore.getRecord(dataKey);
-        return merged && record != null;
+        return merged;
```

Comment:
Record existence should not be checked since it can be removed if a merge policy returns `null`